app/imageswap/imageswap.py

```diff
@@ -97,7 +99,6 @@ def mutate():
     ):
 
         app.logger.info(f'Disable label "{imageswap_disable_label}=disabled" detected for "{workload}" {workload_type}", skipping image swap.')
-        needs_patch = False
```

Comment:
question (blocking):  Is there a reason we wouldn't want to set this here so it short-circuits any additional logic/processing if the disable label is provided?

Reply:
No, it is only clear at this point, that "needs_patch" is already False. So, it has no effect.

Reply:
Restored the assignment at this postition.